docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/indexTemplates.ts

```diff
@@ -0,0 +1,15 @@
+export const appTemplateJs = `
+import { Provider, themes } from "@stardust-ui/react"
+import React from "react";
+import ReactDOM from "react-dom";
+
+import Example from "./example";
+
+const element = document.getElementById("root");
+ReactDOM.render(
+  <Provider theme={themes.teams}>
+    <Example />
+  </Provider>,
+  element
+);
+`.slice(1)
```

Comment:
may suggest to introduce comment for the sake of intent being immediately readable

Reply:
removed this `slice()` at all to avoid confusions